tools/ci/start.sh

```diff
@@ -0,0 +1,39 @@
+# This script is desinged to be sourced from tools/docker/start.sh
+
+# Start userspace OOM killer: https://github.com/rfjakob/earlyoom
+# It will report memory usage every minute and prefer to kill browsers.
+sudo earlyoom -p -r 60 --prefer '(chrome|firefox)' --avoid 'python' &
+
+if [[ ! `git rev-parse --verify -q ${REVISION}` ]];
+then
+    # But if for some reason the commit under test isn't in that range, we give in and
+    # fetch everything
+    git fetch -q --unshallow ${REMOTE}
+    git rev-parse --verify ${REVISION}
+fi
+git checkout -b build ${REVISION}
```

Comment:
If we keep this part in the Docket image, we'll be able to land changes to the new script without fear of affecting pull requests that are based on some commit behind the HEAD of `master`. It will also mean pull requests which modify the new file will be run with the modifications.

Reply:
This was just wrong. Good catch!